File: 01.C#-Homework/07.Advanced CSharp Homework/02.PrimeChecker/PrimeChecker.cs
using System;

class PrimeChecker
{
    static void PrimeCheckerr (long n )
    {
        bool isPrime = true;
        long maxDevider = (long)Math.Sqrt(n);
        if (n ==0 || n==1)
        {
            isPrime = false;
        }
        else
        {
            for (long devider = 2; devider <= maxDevider; devider++)
            {
                if (n % devider == 0 && devider != n)
                {
                    isPrime = false;
                    break;
                }
                else
                {
                    isPrime = true;
                    continue;
                }
            }
        }
        Console.WriteLine(isPrime);

    }
    static void Main()
    {
        long n = long.Parse(Console.ReadLine());
        PrimeCheckerr(n);
    }
}

File: 04.Java Script - Homeworks/02.JavaScript Syntax/04.CheckIfNumberIsPrime/primeChecker.js
function  isPrime(value) {
	var sqareOfVal = Math.round(Math.sqrt(value));
	var prime = false;
	if (value==0 || value==1 || value == 2) {
		prime=false;
	}else if(value>1){
		for (var i = 2; i <=sqareOfVal; i++) {
			if (value%i==0) {
				prime=false;
				break;
			}else{
				prime=true;
				continue;
			}
		}
	}
	return prime;
}
console.log(isPrime(7));
console.log(isPrime(254));
console.log(isPrime(587));
